dockers/docker-snmp/Dockerfile.j2

```diff
@@ -45,6 +45,9 @@ RUN pip3 install --no-cache-dir \
         pyyaml                              \
         smbus
 
+# Install python-redis
+RUN pip3 install redis==4.5.4
```

Comment:
@liuh-80 why do we need this?

Reply:
The swssdk depends on python-redis.
So before we remove swssdk by this PR, python-redis also install to docker-snmp when install swsssdk: https://github.com/sonic-net/sonic-buildimage/pull/11469/files

So to avoid potensial risk that some other script also using python-redis break because swsssdk removed from this docker, maybe we install python-redis to this container.

Reply:
Fixed, code removed.